static/src/js/components/GroupList/GroupList.jsx

```diff
@@ -114,6 +116,12 @@ const GroupList = () => {
     )
   }, [])
 
+  const buildEllipsisTextCell = useCallback((cellData) => cellData && (
+    <EllipsisText>
+      {cellData}
```

Comment:
Thought about this a little more. I wonder if we should build this into EllipsisText component so we don't have to handle it at this level in any component that uses it.